public/index.html

```diff
@@ -17,14 +17,15 @@
 		<!--script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/jquery-ui.min.js"></script-->
 		<link href='http://fonts.googleapis.com/css?family=Oswald' rel='stylesheet' type='text/css'>
 		<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 		<script src="//d3js.org/d3.v4.min.js"></script>
 		<script src="https://code.jquery.com/jquery-migrate-3.1.0.js"></script>
 
     	<script src="/scripts/imageSliderScript.js"></script>
     	<script src="/scripts/imageGridScript.js"></script>
     	<script src="/scripts/timeline.js?version=1005"></script>
-	<script src="/scripts/histogram.js"></script>
+		<script src="/scripts/histogram.js"></script>

```

Comment:
```suggestion
	<script src="/scripts/histogram.js"></script>
```

This is an unnecessary change.